lib/pages/chat_list/chat_list.dart

```diff
@@ -737,6 +744,7 @@ class ChatListController extends State<ChatList>
   @override
   Widget build(BuildContext context) {
     super.build(context);
+    activeRoomId = Matrix.of(context).activeRoomId;
```

Comment:
Why reassign the value here when you already assigned it in `initState()`?

Reply:
because of 2 reasons:
- the `activeRoomId` doesn't update if Navigator is rebuilt
- When the `activeRoomId` changes, it makes the whole `AdaptiveScaffoldRoute` be rebuild

So put it down here solve the problem